lab-kyle/routes/data/notes/123456

```diff
@@ -0,0 +1 @@
+{"id”:”123456”,”content":"test test test"}
```

Comment:
Generally speaking, it is better to gitignore your data/db folder and create anything your program needs at runtime (like a data directory to put stuff in locally).  This will help keep your repository reasonably sized.